src/YoutubeAtom.tsx

```diff
@@ -296,7 +296,12 @@ export const YoutubeAtom = ({
         if (typeof window === 'undefined') return;
 
         if (window.YT) {
-            loadVideo();
+            // An undocumented ready callback to handle edge cases were window.YT.Player may not be ready for init
+            // https://stackoverflow.com/a/62254596
+            // @ts-ignore
```

Comment:
My eslint screams at me for using @ts-ignore... Are we able to get around having to do this? But if we do, do we need to ignore the ignore?

Reply:
so the reason we have this is that `YT.ready` is not offically documented (this the youtube types do not define it). 

It is true that there is therefore a risk that using an undocumented API, I could look at using instead `onYouTubeIframeAPIReady`.

I am particularly concenred of a minor race condition edge cases that might arise if `onYouTubeIframeAPIReady` is called by YT but we have not set any function to that variable.